src/actions/measurements.ts

```diff
@@ -1,16 +1,62 @@
-import { cloneDeep, pick } from "lodash";
+import { cloneDeep } from "lodash";
+import { AppDispatch, ThunkFunction } from "../store";
 import { measurementIdSymbol } from "../util/globals";
-import { defaultMeasurementsControlState } from "../reducers/controls";
+import { ControlsState, defaultMeasurementsControlState, MeasurementsControlState } from "../reducers/controls";
 import { getDefaultMeasurementsState } from "../reducers/measurements";
 import { warningNotification } from "./notifications";
-import {
+import{
```

Comment:
you probably didn't mean to do this?

Reply:
Good eye! Definitely unintended

Reply:
Fixed up in https://github.com/nextstrain/auspice/pull/1910/commits/f5b3f2fb1d744abeade9383ed683219ef98b9cec